### DISCUSSION TITLE
Discord not captures osu properly

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21594] [Q&A]
You can't stream osu because discord gives a black screen with audio behind it.

https://user-images.githubusercontent.com/86934170/206767755-94fa6704-08cc-493c-a879-5cf15a81efa1.mp4

Happened by a chance (1 out of many many tries) but you can still get a normal output:

https://user-images.githubusercontent.com/86934170/206768250-d8a15d43-8e2e-440d-906d-f7d3c60b17e5.mp4

Turning this off doesnt help at all
![image](https://user-images.githubusercontent.com/86934170/206768532-857a0b0d-d825-4f6a-955d-5caa23955c37.png)

The thing is other fullscreen applications work fine. Only osu!lazer and stable doesnt work.

**Comment by KALImaster69** [comment]
any update on this problem op?

**Comment by peppy** [comment]
Works fine here.
